ionic/capacitor.config.json

```diff
@@ -4,5 +4,6 @@
   "bundledWebRuntime": false,
   "npmClient": "npm",
   "webDir": "build",
-  "cordova": {}
+  "cordova": {},
+  "linuxAndroidStudioPath": "/home/jean/android-studio/bin/studio.sh"
```

Comment:
Could we remove the hardcoded username with an env var $HOME, some absolute path like /opt or just remove it since the README mentions how the configure the value.

Reply:
Couldn't use an env var like $HOME, so i remove this config as you suggest. However, i was not able to untrack this file (`git rm --cached <file>`) without deleting it from the distant repo. From now on, we will be careful to not to commit any change to this file.